roles/custom/matrix-bot-draupnir/templates/production.yaml.j2

```
@@ -38,7 +49,7 @@ autojoinOnlyIfManager: true
 
 # If `autojoinOnlyIfManager` is false, only the members in this space can invite
 # the bot to new rooms.
-#acceptInvitesFromSpace: "!qporfwt:example.com"
+#acceptInvitesFromSpace: "!example:example.org"
```

```suggestion
#acceptInvitesFromSpace: "!qporfwt:example.com"
```

This is a common placeholder (despite being difficult to see it) among the documentation.